### PR TITLE
Rub-2: Create Inventory model

### PR DESCRIPTION
**Pull request by mayar2610**
Reviewers:
- [x] @rokor 
- [x] @sonishc 
- [x] @StepanKovalyshyn 
Description:
Create Inventory model, change the base branch from master to develop, delete inventory factory

### CODE REVIEW

File: spec/factories/inventories.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :inventory do
+
```
Comment (rokor) [inline spec/factories/inventories.rb]: Please delete or add some basic fields to factory. Do not leave it empty.

**Review by rokor — CHANGES_REQUESTED** [review]

**Comment by rokor** [comment]
@mayar2610 Target branch should be changed to `develop` not master, rebased and remove all config that already present in branch.

File: spec/factories/inventories.rb (hunk 1 above)
Comment (mayar2610) [reply spec/factories/inventories.rb]: Fixed

**Review by rokor — APPROVED** [review]

**Comment by rokor** [comment]
👍

**Comment by rokor** [comment]
@mayar2610 Also, please add description and 2 more members of the team to the review

**Comment by rokor** [comment]
@mayar2610 Please separate description from header. Header should be only in your case `Rub 2: Create "Inventory" model`. Than in the description it is your first default comment `Create "Inventory" model, change the base branch from master to develop, delete inventory factory`.

**Review by sonishc — APPROVED** [review]
Maybe the method maximum_items_not_more_then_15 has a fairly long name

**Review by StepanKovalyshyn — APPROVED** [review]
method maximum_items_not_more_then_15 can be named inventory_limit (as it shows us the maximum amount of backpack)

**Comment by rokor** [comment]
@mayar2610 Rebase needed. For how to rebase please see my comment to https://github.com/rokor/Ruby_Combats/pull/9

**Pull request merged**